.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: php
+php:
+- 5.3
```

Comment:
"PHP 5.3 is supported only on Precise." - it may be easier just to remove

Reply:
yeah, I wonder if I should maybe keep only 5.6 from the last major version in general

Reply:
I'd keep only 5.6 and up.